lib/cli/scripts/init-acs-env.ts

```diff
@@ -0,0 +1,160 @@
+/* tslint:disable */
+let alfrescoApi = require('@alfresco/js-api');
+let program = require('commander');
+let fs = require ('fs');
+const path = require('path');
+import { logger } from './logger';
+const { SharedlinksApi, FavoritesApi } = require('@alfresco/js-api');
+let MAX_RETRY = 10;
+let counter = 0;
+let TIMEOUT = 6000;
+const ACS_DEFAULT = require('./resources').ACS_DEFAULT;
+/* tslint:enable */
+
+let alfrescoJsApi;
+
+export default async function () {
+    await main();
+}
+
+async function main() {
+
+    program
+        .version('0.1.0')
+        .option('--host [type]', 'Remote environment host')
+        .option('-p, --password [type]', 'password ')
+        .option('-u, --username [type]', 'username ')
+        .parse(process.argv);
+
+    await checkEnv();
+
+    logger.info(`***** Step initialize ACS *****`);
+    await initializeDefaultFiles();
+
+}
+
+async function initializeDefaultFiles() {
+    for (let j = 0; j < ACS_DEFAULT.files.length; j++) {
+        const fileInfo = ACS_DEFAULT.files[j];
+        switch (fileInfo.action) {
```

Comment:
Not sure this will work properly if same file needs to be locked and shared, for example.

Reply:
do we have such a case right now?

Reply:
We don't have such a case right now. As discussed, we can merge as it is and create a separate ticket for improvements